templates/incl-menubar.html

```diff
@@ -38,7 +38,7 @@
     {% if not username %}
       <div class="dropdown inline-block relative {% if not username %}ltr:ml-auto rtl:mr-auto{% endif %} py-2 z-20">
           <button class="bg-blue-400 text-white font-semibold py-1 px-2 rounded inline-flex items-center">
-            <span class="ltr:mr-1 rtl:ml-1">{{ current_language().sym }}</span>
+            <i class="fas fa-fw fa-globe"></i> <span class="ltr:mr-1 rtl:ml-1">{{ current_language().sym }}</span>
```

Comment:
The horizontal distance is now always one whitespace, we can automatically align this nicely horizontally by creating a surrounding `div` with the classes `flex flex-row gap-2`.

Reply:
Ah yeah the globe seems to "float" a bit now vertically? I am not sure why...

Reply:
Adding the `items-center` class shoud do the trick!